core/typeSugar.ml

```diff
@@ -1630,7 +1630,7 @@ let rec close_pattern_type : pattern list -> Types.datatype -> Types.datatype =
 
 type unifyResult = UnifySuccess | UnifyFailure of (Unify.error * SourceCode.pos)
```

Comment:
Yes, types follow the same scheme as terms, so it should really be `unify_result`

Reply:
Done now.